### PR TITLE
nav bar created

### PR DESCRIPTION
**Pull request by YuriMoreau**
- [DEMO LINK](https://YuriMoreau.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://YuriMoreau.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,72 @@
...
+}
+
+header {
+  width: auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 0 50px;
+}
```
Comment (valerii-derkach) [inline src/style.css:12]: use classes as selectors for styling, not tags

File: src/style.css [hunk 2]
```diff
@@ -1 +1,72 @@
...
+  height: 40px;
+  width: 40px;
+  padding: 10px 0;
+  display: inline-block;
+}
+
+ul.nav-links {
+  margin: 0;
+  padding: 0;
+  list-style: none;
```
Comment (valerii-derkach) [inline src/style.css:24]:
```suggestion
nav-links {
  margin: 0;
  padding: 0;
  list-style: none;
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,72 @@
...
+  margin-right: 10px;
+}
+
+.nav-links > :last-child {
+  margin: 0;
+  margin-left: 10px;
+}
+
+.nav-item a {
+  font-family: Roboto, sans-serif;
```
Comment (valerii-derkach) [inline src/style.css:43]:
```suggestion
.nav-item {
  font-family: Roboto, sans-serif;
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,72 @@
...
+  line-height: 15px;
+  padding: 22.5px 0;
+  text-decoration: none;
+  font-size: 12px;
+  color: black;
+  display: inline-block;
+  text-transform: uppercase;
+}
+
+.nav-item a.is-active {
```
Comment (valerii-derkach) [inline src/style.css:53]:
```suggestion
.nav-item .is-active {
```

File: src/style.css [hunk 5]
```diff
@@ -1 +1,72 @@
...
+  position: relative;
+  color: rgba(0, 172, 220, 1);
+}
+
+.nav-item a.is-active:after {
```
Comment (valerii-derkach) [inline src/style.css:58]:
```suggestion
.nav-item .is-active:after {
```

File: src/style.css [hunk 6]
```diff
@@ -1 +1,72 @@
...
+  position: absolute;
+  background-color: rgba(0, 172, 220, 1);
+  width: 100%;
+  height: 4px;
+  left: 0;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+.nav-item a:hover {
```
Comment (valerii-derkach) [inline src/style.css:69]:
```suggestion
.nav-item :hover {
```

**Review by valerii-derkach — CHANGES_REQUESTED** [review]
add demo and test report link and fix comments

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
Please add Demo Link and Test Report Link on your PR description

**Comment by YuriMoreau** [comment]
- [DEMO LINK](https://YuriMoreau.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://YuriMoreau.github.io/layout_moyo-header/report/html_report/)

File: src/style.css [hunk 7]
```diff
@@ -1 +1,71 @@
...
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  padding: 10px 0;
+  display: inline-block;
+}
+
+.nav-links {
```
Comment (stanislav-topikha) [inline src/style.css:21]:
```suggestion
.nav-link {
```

File: src/index.html [hunk 8]
```diff
@@ -6,9 +6,47 @@
...
+          </li>
+          <li class="nav-item">
+            <a class="nav-item-link" href="#">Samsung</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-item-link" href="#">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-item-link" href="#" 
+            data-qa="hover">Ноутбуки и компьютеры</a>
```
Comment (stanislav-topikha) [inline src/index.html:33]:
```suggestion
            <a 
              class="nav-item-link" 
              href="#" 
              data-qa="hover"
            >
              Ноутбуки и компьютеры
            </a>
```

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
add shadow under the header(check the mock)

**Pull request closed without merging**